src/components/Game.js

```diff
@@ -198,6 +200,33 @@ class Game extends Component {
     this.setState({targetTime: null}, () => {devLogger(`in voidTargetTime, targetTime: ${this.state.targetTime}`);});
   }
 
+  // *** FIND NOTE THAT MATCHES USER PITCH ***
+  findUserNoteByPitch(pitch) {
+    let userNote;
+    const availableNotes = this.state.availableNotes;
+    const thisPitch = pitch;
+    // let i = 0;
+
+    if (!pitch) {this.setState({userPlayingNote: null})};
+
+    for (let i = 0; i < availableNotes.length; i+=1) {
+    // while (!userNote) {
+      let thisNoteFreqRangeLower = (availableNotes[i].frequency-(availableNotes[i].frequency*0.02806))
+      let thisNoteFreqRangeUpper = (availableNotes[i].frequency+(availableNotes[i].frequency*0.02973))
```

Comment:
`availableNotes[i]` is accessed 4 times here, can we move it to a variable? 
This for loop can be a `availableNotes.forEach(note => ....` or `userNote = availableNotes.find(note => ...`